modules/misc/news.nix

```diff
@@ -1065,7 +1065,6 @@ in
           A new module is available: 'programs.alacritty'.
         '';
       }
-
```

Comment:
Should keep the empty line to keep a consistent format.

Reply:
Makes sense. Done.